internal/pkg/rpaas/nginx/configuration_render.go

```diff
@@ -127,227 +161,182 @@ var rawNginxConfiguration = `
 # This file was generated by RPaaS (https://github.com/tsuru/rpaas-operator.git)
 # Do not modify this file, any change will be lost.
 
-user {{with .Config.User}}{{.}}{{else}}nginx{{end}};
-worker_processes {{with .Config.WorkerProcesses}}{{.}}{{else}}1{{end}};
+{{- with $config.User }}
+user {{ . }};
+{{- end }}
+
+{{- with $config.WorkerProcesses }}
+worker_processes {{ . }};
+{{- end }}
 
 include modules/*.conf;
 
+{{- template "root" . }}
+
 events {
-    worker_connections {{with .Config.WorkerConnections}}{{.}}{{else}}1024{{end}};
+    {{- with $config.WorkerConnections }}
+    worker_connections {{ . }};
+    {{- end }}
 }
 
-{{template "root" .}}
-
 http {
     include       mime.types;
     default_type  application/octet-stream;
-    server_tokens off;
 
-    sendfile          on;
-    keepalive_timeout 65;
+    {{- if or (not $config.SyslogEnabled) (not (boolValue $config.SyslogEnabled)) }}
```

Comment:
I think the `(not $config.SyslogEnabled)` is not needed as `boolValue` will already return false for a nil pointer.

Reply:
Sure. I'll remove it.